controllers/storagecluster/cephblockpools.go

```diff
@@ -15,19 +15,51 @@ import (
 
 type ocsCephBlockPools struct{}
 
+// ensures that peer cluster secret is exist and adding into cephBlockPool
+func (r *StorageClusterReconciler) addPeerSecretsToCephBlockPool(initData *ocsv1.StorageCluster) (*cephv1.MirroringPeerSpec, error) {
+	secretNames := []string{}
+
+	if len(initData.Spec.Mirroring.SecretNames) == 0 {
+		err := fmt.Errorf("Mirroring peer cluster secretNames not found")
+		return nil, err
```

Comment:
```suggestion
		return nil, fmt.Errorf("Mirroring peer cluster secretNames not found")
```

Reply:
add more information about error. Like "Mirroring is enabled but SecretNames are not provided"